src/modules/expression/binop/or.rs

```diff
@@ -29,12 +47,7 @@ impl SyntaxModule<ParserMetadata> for Or {
     }
 
     fn parse(&mut self, meta: &mut ParserMetadata) -> SyntaxResult {
-        parse_left_expr(meta, &mut self.left, "or")?;
-        let tok = meta.get_current_token();
-        token(meta, "or")?;
-        syntax(meta, &mut *self.right)?;
-        let error = "Logical or operation can only be used on arguments of the same type";
-        expression_arms_of_same_type(meta, &self.left, &self.right, tok, error)?;
+        handle_binop!(meta, "or", self.left, self.right)?;
```

Comment:
```suggestion
        handle_binop!(meta, "disjunction", self.left, self.right)?;
```
or:
```suggestion
        handle_binop!(meta, "disj", self.left, self.right)?;
```

Reply:
"Cannot disjunction value"? How about "Cannot disjoin value"

Reply:
"Cannot disjunction value"? How about "Cannot disjoin value"

```suggestion
        handle_binop!(meta, "disjoin", self.left, self.right)?;
```